test/lit/passes/stack-check-memory64.wast

```diff
@@ -0,0 +1,93 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+;; NOTE: This test was ported using port_test.py and could be cleaned up.
+
+;; RUN: foreach %s %t wasm-opt --stack-check --enable-memory64 -S -o - | filecheck %s
+
+(module
+ (memory i64 (data))
+ ;; CHECK:      (type $none_=>_i64 (func (result i64)))
+
+ ;; CHECK:      (type $i64_i64_=>_none (func (param i64 i64)))
+
+ ;; CHECK:      (global $sp (mut i64) (i64.const 0))
+ (global $sp (mut i64) (i64.const 0))
+ (func "use_stack" (result i64)
+  (global.set $sp (i64.const 42))
+  (global.get $sp)
+ )
+)
+;; CHECK:      (global $__stack_base (mut i64) (i64.const 0))
```

Comment:
@tlively  are these automatically generated checks supposed to sit after the module or inside it?

Reply:
I think it's expected that they're after the module, since they're all that's left when the auto update script has finished matching up outputs with inputs.

It would be nice to improve the auto update script to produce nicer output, but it's already very complex. Rewriting it in a statically typed language rather than python might help manage the complexity and help us extend it.